molgenis-platform-integration-tests/src/test/java/org/molgenis/integrationtest/platform/datatypeediting/StringAttributeTypeUpdateTest.java

```diff
@@ -42,7 +42,8 @@ public void afterClass()
 				{ "A VERY LONG TEXT!!!!@#$#{@}{@}{#%$#*($&@#", TEXT, "A VERY LONG TEXT!!!!@#$#{@}{@}{#%$#*($&@#" },
 				{ "1", ENUM, "1" }, { "<h1>Hello World</h1>", HTML, "<h1>Hello World</h1>" },
 				{ "Compounds go!", COMPOUND, null }, { "1990-11-13", DATE, "1990-11-13" },
-				{ "2016-11-13T20:20:20+0100", DATE_TIME, "2016-11-13T20:20:20+0100" } };
+				{ "2016-11-13T20:20:20+0100", DATE_TIME, "2016-11-13T20:20:20+0100" },
+				{ "2016-11-13T20:20:20+0500", DATE_TIME, "2016-11-13T20:20:20+0500" } };
```

Comment:
Fixes bug, this used to pick system default time zone for all datetime strings when converting to date_time